eng/CredScanSuppression.json

```diff
@@ -90,6 +90,12 @@
                 "sdk/keyvault/Azure.Security.KeyVault.Certificates/tests/SessionRecords/CertificateClientLiveTests/VerifyImportCertificatePemWithoutIssuerAsync.json"
             ],
             "_justification": "Files contain ephemeral keys or certificates used by test code during playback."
+        },
+        {
+            "file":[
+                "sdk/datafactory/Azure.ResourceManager.DataFactory/src/Generated/Models/RestServiceAuthenticationType.cs"
+            ],
+            "_justification": "File contains a string value that ends with credential, not a true credential."
```

Comment:
[CredScan](https://dev.azure.com/azure-sdk/public/_build/results?buildId=1642909&view=logs&j=bc67675d-56bf-581f-e0a2-208848ba68ca&t=7eee3a58-6120-518b-7fcb-7e943712aa81&l=59) failed due to [this file](https://github.com/Azure/azure-sdk-for-net/blob/f06129c225bd15d05ad121871a27f03e6aac4cec/sdk/datafactory/Azure.ResourceManager.DataFactory/src/Generated/Models/RestServiceAuthenticationType.cs#L29), but it's a string value of the enum, not a true secret key.